### ISSUE TITLE
Adding Native Language Terms

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#4779]
[From @sjshirar ]

@AJLinn @camwebb As part of a grant project through the UA Museum archaeology department we are correlating object names and material types with their Inupiaq language counterpart. As part of this project we also want to find a spot within Arctos catalog records to preserve this information. Making this an option within the identification heading along with Scientific Name would be ideal, listed under an Indigenous Name or Ancestral Name heading. We would also like to make this an option as an attribute to provide a place to record all of the metadata associated with each native language term such as who made the determination, when it was made, and what method was used. Here's a rough idea of what the attribute information would look like:

Value: Indigenous term
Unit: [code table culture](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctculture)
Remarks: Description/definition of what the word means
Det. Date: date word provided to museum
Det. Meth: add how you got the word, including things like dictionary search, linguist provided, Indigenous language speaker, other publication source, etc.
Determiner: agent who provided you the name (person, organization, author of publication, etc.

We look forward to hearing comments and ideas about the best way to move forward with this.
Thanks!

**Comment by dustymc** [comment]
Maybe an opportunity to do something with [Jonathan Amith](https://github.com/ArctosDB/new-collections/issues/252)?

Attributes should be relatively easy.

https://github.com/ArctosDB/arctos/issues/3540 would allow multiple equally-valid identifications. Ideally that would be accompanied by some sort of Inupiaq (and/or whatever) taxonomy, but just A-string IDs would get the idea across.

https://github.com/ArctosDB/arctos/issues/3913 might somehow provide a bigger-picture approach.

**Comment by sjshirar** [comment]
If we can get the attribute created as soon as possible that would be great!

https://github.com/ArctosDB/arctos/issues/3540 does seem along the same line, but here we would need the identification labeled differently (not listed as scientific name or even nested under scientific name). Is that a possibility? @AJLinn Angie and I have discussed developing a taxonomy but that it would be a major undertaking more suited for a future grant funded project, if it's even realistic at all given the number of languages and different dialects. For right now we'd like to keep it as free-text (A-string).

Regarding https://github.com/ArctosDB/arctos/issues/3913 I'm not aware of software that translates Alaska Native languages. If it does exist we'd need to test its accuracy. Right now we're working directly with speakers to get the translations.

**Comment by Jegelewicz** [comment]
>     not listed as scientific name
> 
> New Issue, I think - that should probably be globally changed (perhaps even including structurally) to "identified as" or something of the sort, but that would need discussion.

I could see two things happening here.
1. For native names associated with formal taxa, this could be treated as a common name BUT we need a way to indicate the language for common names (probably something we should do anyway?
2. For native names associated with the {string} part of an identification, why not add another identification where the string is the native term? Indicate the language in the identification determination method or identification remark.

**Comment by Jegelewicz** [comment]
> Probably worth looping Jonathan (mentioned above - @Jegelewicz do you have contact info??) in on that - eg, sounds like that might be best represented as audio files rather than/in addition to/???? text (and probably transliterated text, and...)

https://github.com/ArctosDB/internal/issues/54

**Comment by dustymc** [comment]
> only have one "accepted"

We have a [solution](https://github.com/ArctosDB/arctos/issues/4779#issuecomment-1160881340), just needs discussed/implemented. (Or I certainly have no objections to the the astring-->"standard" taxonomy approach, it's just a bit less formal/structured.)

**Comment by Jegelewicz** [comment]
I think this needs to go through @ArctosDB/arctos-code-table-administrators  and/or @ArctosDB/taxonomy , but may also need a focus group to ensure we handle it the best possible way. Also, waiting on: [Allow variably ranked acceptedness of identifications #3540](https://github.com/ArctosDB/arctos/issues/3540)

**Comment by AJLinn** [comment]
I think @sjshirar was hoping to discuss this at the upcoming issues meeting. There is NSF funding on the table right now to implement this so there is some urgency. What can I do to help facilitate moving it forward?

**Comment by Jegelewicz** [comment]
@AJLinn the problem here is that there is no real proposal. When things like this get brought up at the issues meeting, they tend to take up most of the time, then things that can be easily resolved never get done. I think that we need to discuss this in a smaller committee and bring a proposal to the community. As I see it, the first question that needs answering is: 

Are we proposing to add native language terms to the taxon names table, as attributes, or as alternate names for things in the taxon names table?

I am not opposed to discussing this at the issues meeting - but I'd like to see it discussed after all of the committee issues. The committees are meeting monthly to resolve issues so that the community can make a quick decision and I'd like to make sure that their work is getting some resolution before we go on to things that are still in the experimental stage.

I hope that makes sense!

**Comment by AJLinn** [comment]
>  there is no real proposal. 

This is exactly what we're frustrated about. We have attempted to make proposals which were met with roadblocks and a lack of engagement for options, or options that wanted to invite external people into a process that we already have moving ahead and have a vision for, in order to improve the accessibility and diversity of the knowledge represented in our records.

@Jegelewicz @sjshirar @camwebb @dustymc and I should have a meeting (apparently not an issues meeting even though this is an existing issue that has implications for other collections) ASAP to discuss what we want to do, what roadblocks exist for implementing as we want to see done, and what we need to do to remove the roadblocks.

Please advise/provide options for meeting. Scott, Cam, and I are all physically located in offices directly next to each other so Teresa and Dusty would need to join us remotely.

To review, this was the original proposal to get the words and metadata into the records.

> NEW ATTRIBUTE
> Value: Indigenous term
> Unit: [code table culture](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctculture)
> Remarks: Description/definition of what the word means
> Det. Date: date word provided to museum
> Det. Meth: add how you got the word, including things like dictionary search, linguist provided, Indigenous language speaker, other publication source, etc.
> Determiner: agent who provided you the name (person, organization, author of publication, etc.
> 

A second option was provided to us with a potential future shift to identifications, but **only** if/when multiple accepted identifications are available to be displayed at once. 
Personally, while I love the idea Scott brought up about creating culturally-specific taxonomies as a way of decolonizing our collections, we are not there yet and won't be for a very long time and a lot of money, unless someone new comes along and does the work for us. 
Following the attribute model while also adding the Indigenous term to the A {string} using the existing taxonomy in place for cultural collections achieves our two goals:
1) adding culturally-specific terms to objects in our collection with existing cultural designations
2) adding metadata associated with the terminology that can be traced to who provided that information (an agent whose language and cultural information will be outlined in their agent record)

Please advise & recommend the next step. As I mentioned in the original proposal and comment from yesterday, there are existing and future NSF projects funding this work.

**Comment by Jegelewicz** [comment]
Let's start by picking a meeting day/time. Please indicate your availability in [this When2Meet](https://www.when2meet.com/?18140042-NwxKu).

I think there may be others who might like to participate, so we should announce the When2Meet in the issues meeting.

**Comment by AJLinn** [comment]
> FWIW, I believe that the taxonomy route will be the best path and we have already approved https://github.com/ArctosDB/arctos/issues/3540. It just needs implementing.

The inherent problem with this approach as I see it, even with multiple accepted identifications, is that necessarily only one of these multiple IDs will be displayed in the search results page. Right now, as I showed above with my mask records, we put the Indigenous term in the A{string} after a slash, so the term is seen in the results alongside the English term. If we go the taxonomy route, as I can see it, we will still have to do what we're currently doing, PLUS adding the same information to the A{string}. 

But I'm happy to hear what @camwebb and @sjshirar think since we are the ones trying to implement this now.

**Comment by Jegelewicz** [comment]
@AJLinn is this essentially a duplicate of #4805 ?

**Comment by AJLinn** [comment]
Yes, thanks for linking them. I filed this one first and then Scott followed up with the second.

**Comment by sjshirar** [comment]
I think at this point moving forward with an attribute and adding native language terms to the existing A-string identifications is the way to go, unless the issue of how search results are shown can be addressed in a timely manner. I don't know enough about what that change would entail. We're at a point in our grant where we have Iñupiaq terms to input, so the route that would get us entering that information sooner is how we'd like to proceed.

**Comment by dustymc** [comment]
Including Arctos in the funding would have provided a mechanism to discuss and a clear priority. @mkoo maybe that needs somehow clarified in the administrative stuff??

I agree that an Issues meeting isn't the best place to get started. Those discussions should be limited to "here's what we plan, please tell us how we've got it wrong." A "focus group" to get to that stage seems useful.

>necessarily only one of these multiple IDs will be displayed in the search results page

No! I'm not sure what we're going to do with that, but https://github.com/ArctosDB/arctos/issues/3540#issuecomment-1146081349 provides a starting point and should be seen as a plea for help. (And allowing ANY UI to drive data is just bass-ackwards....)

>taxonomy route

I think maybe there's some misunderstanding. AFAIK taxonomy is solid and can do whatever you might want now. An identification using a not-as-taxonomy term can still use any taxon+classification. I'm not sure where this comment came from or what's being suggested, but we should all be clear on what's what before moving on.

ANYWAY, assuming I understand anything:

* Identifications seem clearly "correct" for this, but do require some development
* Attributes can be duct-taped to all sorts of strange things, and might at least provide a safe place to stash some data until something can be done with identifications - unless there are multiple IDs for whatever reason, in which case the attributes are likely to be lost in the jumble.

**Comment by AJLinn** [comment]
> ANYWAY, assuming I understand anything:
Identifications seem clearly "correct" for this, but do require some development

Okay, I'm done asking for permission here. 
Based on the people who are experts in this subject area and the information that we have already provided (Scott's example indicating that IDs of object names are NOT the only thing being identified with Indigenous terminology, but could include material names, cultural context, site names, etc.), I am now insisting that we create an attribute called Indigenous terminology with free text in each of the fields (like description) and that it be applied to EH, Arc, and Art collections. 

I'm done with hashing this out over Github thru written comments where you are not listening to us, as content experts, who understand the nuances and complications of what we are trying to accomplish and instead are trying to interpret what we are "trying to do" based on your understanding of biological data models.

As we have been told over and over, attributes is where we can create unique elements to describe the individual thing that is cataloged. Indigenous terminology for objects, materials, colloquialisms, stories, are **unique** to a person, a family, a community, a region and the individual piece based on the specific context of THAT object. ONLY with the direction of a Native language speaker / cultural expert are those names applied more broadly to a group of items (e.g., not all Yup'ik masks are kegginaquq, some are agaya, some are nepcetaq). Our "normalized" data model/structure that you are attempting to force upon us is not right and is counter to the work we in the cultural collections are attempting to do to honor the knowledge systems of the people whose collections are are tasked with caring for - this includes their intellectual knowledge that is unique to them.

> Including Arctos in the funding would have provided a mechanism to discuss and a clear priority. @mkoo maybe that needs somehow clarified in the administrative stuff??

UAM already pays Arctos one of the highest user fees to do this basic work on our collection behalf. We have NOT asked for the complicated changes you think we need. Please create the attribute "Indigenous terminology" and make it usable for EH, Arc, and Art so we can proceed with the work we asked for six months ago.
